src/Microsoft.Extensions.DependencyInjection/DefaultServiceProviderFactory.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public class DefaultServiceProviderFactory : ServiceProviderFactory<IServiceCollection>
```

Comment:
So this will be registered by hosting by default? I'm assuming we won't have an extension method like `WebHostBuilder.UseAutofac` for this.